src/maintainer/conda_forge_yml.rst

```diff
@@ -133,7 +135,7 @@ long-term support (LTS) branch for a package.
 build_platform
 --------------
 This is a mapping from the target platform to the build platform for the package
-to be built. e.g. the following builds a ``osx-64`` package on the ``linux-64``
+to be built. e.g. the following builds a ``osx_64`` package on the ``linux_64``
```

Comment:
Why the change to underscores? I would swear `build_platform` uses dashes, like every other conda subdir string.